tests/unit/tasks/test-test.js

```diff
@@ -107,4 +107,31 @@ describe('test task test', function () {
       sslCert: 'ssl/server.cert',
     });
   });
+
+  it('supports properly disabling SSL by nullifying related keys', function () {
```

Comment:
This test passes without the code changes as well, which makes it hard to tell if these changes are desired.
Also, the expectations in this test are already covered here:
https://github.com/ember-cli/ember-cli/blob/master/tests/unit/tasks/test-test.js#L55-L56